### PR TITLE
allow version 0.14.8 of rector/rector

### PR DESCRIPTION
**Pull request by Kanti**

### CODE REVIEW

**Comment by sabbelasichon** [comment]
@Kanti Thanks. But we have the policy to freeze rector version till patch level. Otherwise can not claim to provide a stable release of ssch/typo3-rector. Could you change it to 0.14.8 please. But let´s see what the tests are saying.

**Comment by sabbelasichon** [comment]
@Kanti Tests are green. Awesome. Let´s freeze it.

File: composer.json [hunk 1]
```diff
@@ -28,7 +28,7 @@
         "nette/utils": "^3.0",
         "nikic/php-parser": "^4.14",
         "phpstan/phpstan": "^1.7.12",
-        "rector/rector": "0.14.7",
+        "rector/rector": "^0.14.7",
```
Comment (Kanti) [inline composer.json:31]:
```suggestion
        "rector/rector": "0.14.8",
```

**Comment by sabbelasichon** [comment]
@Kanti New release is also published https://github.com/sabbelasichon/typo3-rector/releases/tag/v1.0.3. Have fun.

**Pull request merged**